docs/deploying/index.md

```diff
@@ -0,0 +1,48 @@
+# Deploying Overview
+
+SpacetimeDB supports both hosted and self-hosted publishing in multiple ways. Below, we will:
+
+1. Generally introduce Identities.
+1. Generally introduce Servers.
+1Choose to proceed with either a [Hosted](/docs/deploying/hosted.md) or [Self-Hosted](/docs/deploying/self-hosted.md) deployment.
```

Comment:
typo? should it be `1. Choose` or just `Choose`?

Reply:
> typo? should it be `1. Choose` or just `Choose`?

`1. Choose` - fixing >>